packages/notification-center/src/i18n/languages/mr.ts

```diff
@@ -0,0 +1,11 @@
+import { ITranslationEntry } from '../lang';
+
+export const HR: ITranslationEntry = {
```

Comment:
```suggestion
export const MR: ITranslationEntry = {
```

Just noticed that the export is wrong here :)